api/schema/definition.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Definition",
+  "type": "object",
+  "properties": {
+    "title": {
+      "type": "string"
+    },
+    "metadata": {
+      "type": "object",
+      "properties": {
+        "@type": { "type": "string" },
+        "language": { "type": "string" },
+        "modified": { "type": "string" },
+        "name": { "type": "string" },
+        "title": { "type": "string" }
+      }
+    },
+    "definitions": {
+      "type": "array",
+      "items": {
+        "type": "object"
+      }
+    },
+    "links": {
+      "type": "array",
+      "items": {
+        "type": "object"
+      }
```

Comment:
I can't figure out how to make it so that it knows what the structure of an item in the `links` array looks like. Same issue as a "definition" object in the `definitions` array.